composer.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "swedbank-pay/magento2-payments",
```

Comment:
Since there's no version number in the `composer.json` file, I assume the version number will be set by a Git tag? Can we include a `beta` label in that for now?

Reply:
@asbjornu Yeah we can tag it instead of versioning in the `composer.json`